asistente_ladm_col/asistente_ladm_col_plugin.py

```diff
@@ -1042,7 +1043,7 @@ def call_fill_right_of_way_relations(self, *args):
     @_validate_if_wizard_is_open
     @_qgis_model_baker_required
     @_db_connection_required
-    @_survey_model_required
+    @_survey_model_and_cadastral_cartography_model_required
```

Comment:
Why not adding another decorator here, so we end up with a decorator for survey and another for cadastral cartography, as we are used to?